admin/load.php

```diff
@@ -636,3 +652,63 @@ function perflab_plugin_admin_notices() {
 	}
 }
 add_action( 'admin_notices', 'perflab_plugin_admin_notices' );
+
+/**
+ * Callback function that print plugin activation script.
+ *
+ * @since n.e.x.t
+ */
+function perflab_print_plugin_activation_script() {
+	$js = <<<JS
+( function( $ ) {
+	$( document ).ajaxComplete( function( event, xhr, settings ) {
+		// Check if this is the 'install-plugin' request.
+		if ( settings.data && typeof settings.data === 'string' && settings.data.includes( 'action=install-plugin' ) ) {
+			// Split the data string by '&' to separate parameters.
+			var params = settings.data.split( '&' );
+
+			// Loop through the parameters to find the 'slug' parameter.
+			var slug = '';
+			for ( var i = 0; i < params.length; i++ ) {
+				var param = params[ i ].split( '=' );
+				if ( param[0] === 'slug') {
+					slug = param[1];
+					break;
+				}
+			}
```

Comment:
If I understand the `settings.data` correctly, this can be simplified by using `URLSearchParams`:

```suggestion
			var params = new URLSearchParams( settings.data );
			var slug = params.get('slug');
```